libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32H743xx.py

```diff
@@ -30,7 +30,7 @@
         (0x20000000, 128, 2), # DTCM, tightly coupled, no DMA, fast
         (0x24000000, 512, 4), # AXI SRAM.
 		(0x00000400,  63, 2), # ITCM (first 1k removed, to keep address 0 unused)
-        (0x30040000,  32, 0), # SRAM3.
+        #(0x30040000,  32, 0), # SRAM3.
```

Comment:
Should this be commented or removed?

Reply:
need to use SDMMC approach to DMA alloc for ethernet